src/main/java/org/jboss/logmanager/FileMove.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.jboss.logmanager;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ *
+ * @author panos
+ */
+public class FileMove {
+    public static void move(File afile, File bfile) {
+        InputStream inStream = null;
+	OutputStream outStream = null;
+		
+    	try{		
+    	    inStream = new FileInputStream(afile);
+    	    outStream = new FileOutputStream(bfile);
+        	
+    	    byte[] buffer = new byte[1024];
+    		
+    	    int length;
+    	    //copy the file content in bytes 
+    	    while ((length = inStream.read(buffer)) > 0){
+    	  
+    	    	outStream.write(buffer, 0, length);
+    	 
+    	    }
+    	 
+    	    inStream.close();
+    	    outStream.close();
+    	    
+    	    //delete the original file
+    	    afile.delete();
+    	    
+    	}catch(IOException e){
+    	    e.printStackTrace();
```

Comment:
This should just throw an `IOException` and force the consumer to deal with the exception.